config/patches/ncurses/config_guess_2015-09-24.patch

```diff
@@ -0,0 +1,857 @@
+--- config.guess.orig	2010-09-28 17:11:39.000000000 -0600
++++ config.guess	2015-10-16 12:51:43.293305861 -0600
+@@ -1,14 +1,12 @@
+ #! /bin/sh
+ # Attempt to guess a canonical system name.
+-#   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
+-#   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010
+-#   Free Software Foundation, Inc.
++#   Copyright 1992-2015 Free Software Foundation, Inc.
+ 
+-timestamp='2010-09-24'
++timestamp='2015-09-14'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
+-# the Free Software Foundation; either version 2 of the License, or
++# the Free Software Foundation; either version 3 of the License, or
```

Comment:
I'm still fairly concerned that this is an issue, should get an actual lawyers advice particularly since it looks like its GPLv3 and not LGPLv3, so its inherently more 'infectious'.